pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -184,6 +185,17 @@ def implements(self, interface=None):
     
     def name(self):
         return self.opts.get('name', None)
+
+    def setClickable(self, s, width=None):
+        self.curve.setClickable(s, width)
+
+    @property
+    def clickable(self):
+        return self.curve.clickable
+
+    @clickable.setter
```

Comment:
Do we need both the `setClickable` method and the property setter? In general I have avoided setters in pyqtgraph, in part to be consistent with Qt, and also to avoid the issue where setting a misspelled property is silently ignored.